### PR TITLE
Updated scripts for CentOS compatibility

### PR DESCRIPTION
**Pull request by troplolBE**
**What is this PR for?**
I make this small PR to propose some changes I made in the helk_install and helk_remove_containers scripts. the biggest changes are in the remove script because I saw that the script doesn't remove everything. To make the script remove the containers, the images, the volumes and the network, I decided to make use of the docker-compose command.

I also added a small script to perform firewall editing on centos machines. Modifying iptables is not enough to open up ports.

**What type of PR is it?**
[Bug Fix]

**How should this be tested?**
Try to install HELK on a CentOS 7 machine and then remove it using the new helk_remove_containers.sh script.

**Questions:**
* Do the licenses files need update? No
* Are there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docker/helk_install.sh [hunk 1]
```diff
@@ -21,11 +21,12 @@ INSTALL_MINIMUM_MEMORY=5000
 ## In MBs
 INSTALL_MINIMUM_MEMORY_NOTEBOOK=8000
 ## In GBs
-INSTALL_MINIMUM_DISK=20
+INSTALL_MINIMUM_DISK=25
```
Comment (neu5ron) [inline docker/helk_install.sh]: need wiki update if this is increased to 25. also, any reason from 20 to 25? I am OK with this but just wanted to know for sure.

File: configs/firewalld/helk.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>helk</short>
+  <description>Firewall service for the HELK.</description>
+  <port protocol="tcp" port="9092" />
```
Comment (neu5ron) [inline configs/firewalld/helk.xml]: we should add ports for nginx/juypter and logstash and other kafka port if it ever gets used:
ports as follows:
nginx:
443
8088
logstash:
3515
5044
8531
kafka:
9092
9093

File: docker/helk_install.sh (hunk 1 above)
Comment (troplolBE) [reply docker/helk_install.sh]: No there is no particular reason. It is a mistake of mine. I will fix that later tonight.

**Comment by neu5ron** [comment]
other then my few comments, looks pretty good - one question, nothing needs done for nginx to expose port 443? such as `setsebool httpd_can_network_connect 1 -P`
thanks for all of this! this is going to save us a ton of time having to figure out everything related to CENTOS (its been a little while for me having used it)

File: configs/firewalld/helk.xml (hunk 2 above)
Comment (troplolBE) [reply configs/firewalld/helk.xml]: I will add all these ports but I think I will work on a newer version of the script with different versions of the service. Like this we will prevent ports from being open for nothing ;)

**Comment by troplolBE** [comment]
I have to admit that I haven’t investigated for that but I will start with that right after the 2 promised fixes.

**Comment by neu5ron** [comment]
take your time, I am off on and anyways.
also join the slack channel so easier to communicate
https://launchpass.com/threathunting

**Comment by troplolBE** [comment]
I will also add some informations into the wiki about centos. Also for the installation script modified on my repo, I installed HELK with it 3 times and it went well the 3 times. I will perform further tests on different versions of centos during the rest of the week.

**Comment by neu5ron** [comment]
fantastic, thanks! I will be around this weekend, so I should have some time to merge this in

**Comment by neu5ron** [comment]
@troplolBE check out the merge and changes I made here:
https://github.com/neu5ron/HELK/tree/scripts-helk_install_and_update

let me know if you can test today/tomorrow

**Pull request merged**